model/src/main/kotlin/com/android/gradle/replicator/model/internal/resources/ResourceData.kt

```diff
@@ -0,0 +1,87 @@
+package com.android.gradle.replicator.model.internal.resources
+
+// Supported file types of each resource. Not exhaustive.
+val ANDROID_RESOURCE_FOLDER_CONVENTION = mapOf(
```

Comment:
same as previous CL, would be nice to get all these constants loaded from a file instead of hardcoded in the code. users could have specific needs.

Reply:
Should this be a user accessible config file?

What I'm thinking is java resource with default values, gets copied by the project generator into the project root where the user can modify it. It then gets picked up by the resource generator.

How's that sound?

Reply:
Also, might be better to do this in another pull request. I can have this done in a day.